src/libs/actions/Search.ts

```diff
@@ -84,6 +88,23 @@ function holdMoneyRequestOnSearch(hash: number, transactionIDList: string[], com
     API.write(WRITE_COMMANDS.HOLD_MONEY_REQUEST_ON_SEARCH, {hash, transactionIDList, comment}, {optimisticData, finallyData});
 }
 
+function areSelectedRequestsUnsubmitted(selectedTransactionIDs: string[]) {
```

Comment:
This function, even though called from Search-adjacent views is not at all connected to search. I don't understand all the details, but it seems like it checks whether reports are in some specific state.
So
 - it should not be inside `actions`
 - it should not be inside `Search***` but rather either Reports or something related to CSVexporting?

Reply:
Agreed, I've moved it into `TransactionUtils` since it seems more fitting for this function.